build.gradle

```diff
@@ -1,36 +1,11 @@
 // Top-level build file where you can add configuration options common to all sub-projects/modules.
 buildscript {
-    ext {
-        kotlin_version = '1.7.10'
-        // KSP releases need to match the Kotlin version - https://github.com/google/ksp/releases
-        ksp_version = "$kotlin_version-1.0.6"
-        hilt_version = '2.43.2'
-    }
-    repositories {
-        google()
-        mavenLocal()
-
-        maven {
-            url 'https://a8c-libs.s3.amazonaws.com/android' 
-            content {
-                includeGroup "com.automattic.android"
-                includeGroup "com.automattic.android.publish-to-s3"
-            }
-        }
-    }
```

Comment:
From the blank Android project and examples I have seen online it seems like you define this in your `settings.gradle` file instead.